src/components/cards/ValidationList/ValidationList.tsx

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useState } from 'react';
+
+import { ValidationObject } from '../../../types';
+import { AlertCircle, CircleCheck } from '../../icons';
+import { Tooltip } from '../../layout';
+
+function ValidationList({ validations }: { validations: ValidationObject[] }) {
+  const [validationChecklist, setValidationChecklist] =
+    useState<Array<JSX.Element | undefined>>();
+
+  useEffect(() => {
+    if (validations) {
+      updateValidationList(validations);
+    }
+  }, [validations]);
+
+  function updateValidationList(validations: ValidationObject[]) {
+    const newElements = validations.map(
+      (validationItem: ValidationObject, index: number) => {
+        console.log(validationItem.status);
+
+        switch (validationItem.status) {
+          case true: {
+            return (
+              <>
+                <Tooltip key={index} message={'Success!'}>
+                  <span className="flex flex-row text flex-left white bold">
+                    <CircleCheck
+                      width={20}
+                      height={20}
+                      fill={'var(--success-green)'}
+                    />
+                    {validationItem.name}
+                    <AlertCircle
+                      width={'16px'}
+                      height={'16px'}
+                      fill={'var(--text-white)'}
+                    />
+                  </span>
+                </Tooltip>
+              </>
+            );
+          }
+
+          case false: {
+            return (
+              <>
+                <Tooltip key={index} message={validationItem.error}>
+                  <span className="flex flex-row text flex-left white bold">
+                    <AlertCircle
+                      width={20}
+                      height={20}
+                      fill={'var(--error-red)'}
+                    />
+                    {validationItem.name}
+                    <AlertCircle
+                      width={'16px'}
+                      height={'16px'}
+                      fill={'var(--text-white)'}
+                    />
+                  </span>
+                </Tooltip>
+              </>
+            );
+          }
+          default:
+            return (
+              <>
+                <Tooltip key={index} message={validationItem.status}>
+                  <span className="flex flex-row text flex-left white bold">
+                    <AlertCircle
+                      width={20}
+                      height={20}
+                      fill={'var(--text-faded)'}
+                    />
+                    Invalid Status Code
+                    <AlertCircle
+                      width={'16px'}
+                      height={'16px'}
+                      fill={'var(--text-white)'}
+                    />
+                  </span>
+                </Tooltip>
```

Comment:
do we need the tool tip here? any input from lucas on this design wise?

Reply:
Still discussing this with lucas but I mentioned and he seemed cool with it - dont have a solid answer yet.

Reply:
it appears noisy and the check itself indicates if it passed/failed so feel it's redundant